### DISCUSSION TITLE
Postgres Installation Keeps Checking Requirements

### DISCUSSION THREAD

**Discussion by stubevan** [Kovah/LinkAce#591] [Support]
Hi,

A fresh (non-docker) installation using postgres as the DB.

.env is correctly setup (DB_CONNECTION=pgsql, DB_PORT=5432) with a remote database

artisan migrate successfully run, and there are tables in the database

registeruser has been run (and I can see the entry in the db)

But when you access the site it always goes to the check requirements page (and it needed the __pdo_mysql__ driver to pass the test) to get this correct and when you try and set up the database you get 

```[2022-12-26 14:44:22] production.ERROR: SQLSTATE[HY000] [2006] MySQL server has gone away ```

in the error log.

It feels as if it's ignoring the DB_CONNECTION entry in the env file but it is being read as if you set that to a non sensical value it errors.

Thanks

**Comment by Kovah** [comment]
Hi, could you please try this: https://github.com/Kovah/LinkAce/issues/517#issuecomment-1192209957

**Reply by stubevan** [reply]
Thanks for the swift reply 

Yes .. which didn't address it

But I solved it (temporarily) by changing line 18 of ./config/database.php to be

```    'default' => env('DB_CONNECTION', 'pgsql'), ```

I love linkace.   I'm trying to move away from docker because I'm running it on a TrueNAS appliance, and I've been having performance problems with the Linux virtualisation (not your problem!).  When running in a FreeBSD jail - it's just peachy!

Thanks
